renderer/suncg/scene.cc

```diff
@@ -46,6 +48,13 @@ in vec3 normal;
 in vec2 texcoord;
 out vec4 fragcolor;
 
+// Note these values need to match DEFAULT_NEAR and DEFAULT_FAR in camera.h
+const float NEAR = 0.1f;
```

Comment:
Hm... does this mean that this will introduce inconsistencies if we ever change the near and far planes in the camera? https://github.com/facebookresearch/House3D/blob/master/renderer/gl/camera.hh#L35

They are public / settable parameters right now, it seems.  This seems like it was a pre-existing issue, though

Reply:
Yes, it was pre-existing and I noticed it, too.